client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml

```diff
@@ -88,6 +96,11 @@ spec:
                 oneOf:
                 - required: ["snapshotHandle"]
                 - required: ["volumeHandle"]
+              sourceVolumeMode:
```

Comment:
Have you included all the changes generated by running the script?  I wonder why lines 75-165 (except for this new sourceVolumeMode) do not have indentation changes.

Reply:
Running the script actually removes the entire section and replaces it with:
```
          metadata:
            type: object
```

So I had to manually copy these files back ..